src/pages/patterns/loading-pattern/index.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Loading
+---
+
+<PageDescription>
+
+Loading patterns are used when information takes an extended amount of time to process and appear on screen. Skeleton states and the loading component are two interactions that communicate to users that data is loading and the screen is not frozen.
+
+</PageDescription>
+
+## Skeleton states
+
+Skeleton states are simplified versions of components used on an initial page load to indicate that the information on the page has not fully loaded yet. They should only appear for one to three seconds, disappearing once the actual content populates the page.
```

Comment:
Potential rephrase / missing points:
- Purpose of skeleton state is to improve a webapp's perceived performance by create a first meaningful "paint". This creates an impression of the app loading faster than it actually does
- Skeleton state helps reduce jumping of a page layout as data loads in.

Reply:
Awesome, added the first point. Thank you!

I don't think the second point is necessary for this section.